tests/ops/mapper/test_generate_caption_mapper.py

```diff
@@ -0,0 +1,121 @@
+import os
+import unittest
+
+from datasets import Dataset
+
+from data_juicer.ops.mapper.generate_caption_mapper import \
+    GenerateCaptionMapper
+from data_juicer.utils.mm_utils import SpecialTokens
+
+
+class GenerateCaptionMapperTest(unittest.TestCase):
```

Comment:
You can erase the hf model after this unit test is finished using DataJuicerTestCaseBase and customized tearDownClass behavior.

Ref: 
https://github.com/alibaba/data-juicer/blob/c97aab3e0330438c435920093b44045ed87c53c4/tests/ops/filter/test_image_text_matching_filter.py#L9C1-L23